src/repr/src/row.rs

```diff
@@ -2378,6 +2379,49 @@ impl Default for RowArena {
     }
 }
 
+/// A thread-local row, which can be borrowed and returned.
+/// # Example
+///
+/// Use this type instead of creating a new row:
```

Comment:
Could you extend this comment to also say why this is preferable to creating a new row?